apps/opik-frontend/src/api/prompts/usePromptsList.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryFunctionContext, useQuery } from "@tanstack/react-query";
+import api, { PROMPTS_REST_ENDPOINT, QueryConfig } from "@/api/api";
+import { Prompt } from "@/types/prompts";
+
+type UsePromptsListParams = {
+  workspaceName: string;
+  search?: string;
+  page: number;
+  size: number;
+};
+
+type UsePromptsListResponse = {
+  content: Prompt[];
+  total: number;
+};
+
+const getPromptsList = async (
+  { signal }: QueryFunctionContext,
+  { workspaceName, search, size, page }: UsePromptsListParams,
+) => {
+  const { data } = await api.get(PROMPTS_REST_ENDPOINT, {
+    signal,
+    params: {
+      workspace_name: workspaceName,
```

Comment:
No need of `workspace_name` here, it's okay for the dependency but no need in the endpoint